support/azure/azure-kubernetes/availability-performance/container-image-pull-performance.md

```diff
@@ -16,7 +16,9 @@ This article provides methods to improve container image pull performance in Azu
 
 There are two types of container image pulls: serialized and parallel image pulls.
 
-In AKS versions earlier than 1.31, AKS enables serialized image pulls by default. Typically, serialized image pulls should be less performant than parallel image pulls. Starting from AKS version 1.31 preview, AKS pulls container images in parallel by default. This change is intended to improve container image pull performance, especially when you deal with large or numerous container images. However, you might still experience decreased performance compared to using serialized image pulls.
+In AKS versions earlier than 1.31, AKS enables serialized image pulls by default. Starting from AKS version 1.31 preview, AKS pulls container images in parallel by default. Typically, serialized image pulls should be less performant than parallel image pulls, particularly when attempting to pull large or numerous container images.
```

Comment:
In AKS versions earlier than 1.31, by default, AKS enables serialized image pulls. Starting in AKS version 1.31 preview, by default, AKS pulls container images in parallel. Typically, serialized image pulls should be less performant than parallel image pulls, particularly when the service tries to pull large or numerous container images.